app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -45,10 +47,12 @@ import kotlin.collections.ArrayList
 
 class ProjectInformationActivity : AppCompatActivity() {
 
+
+    //TODO :  do a PR to correct the apply button code and the favorite button code together
     companion object {
-        val LOADING_STRING = "LOADING"
-        val APPLY_STRING = "APPLY"
-        val UNAPPLY_STRING = "UNAPPLY"
+        const val LOADING_STRING = "LOADING"
+        const val APPLY_STRING = "APPLY"
+        const val UNAPPLY_STRING = "UNAPPLY"
```

Comment:
These could be string resources

Reply:
Yup, this was not my code so I was reluctant to change it but it's done